packages/code-du-travail-api/tests/create_indexes.js

```diff
@@ -36,7 +38,7 @@ async function createIndex(indexName, mappings, data) {
             }
           }
         },
-        mappings: mappings
+        ...(mappings && { mappings })
```

Comment:
does passing null or undefined not worked ? also it could be a good idea to add a simple mapping

Reply:
This is because I have not made any mapping for ccn yet. Not sure how it should look like